.github/workflows/check-documentation-build.yml

```diff
@@ -0,0 +1,51 @@
+name: Documentation Check
+
+on:
+  push:
+    branches: 
+      - "main"
+    paths:
+      - "docs/**"
+      - ".readthedocs.yaml"
+      - "setup.py"
+      - ".github/workflows/check-documentation-build.yml"
+
+  pull_request:
+    branches:
+      - "main"
+      - "stable"
+    paths:
+      - "docs/**"
+      - ".readthedocs.yaml"
+      - "setup.py"
+      - ".github/workflows/check-documentation-build.yml"
```

Comment:
Should we add `VERSION` to this list, in case it's been a long time since the workflow ran before a release?